ovn-fake-multinode-utils/get-config-value.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
This is just a habit of mine. We can use either, but using python3 ensures that we'll use python 3 instead of 2. Hopefully nobody is defaulting to python 2 at this point though.

Reply:
If you feel like changing the other occurrences too that would be great.  Otherwise, no big deal, I can do it in a follow up PR.  Just let me know what you prefer, thanks!

Reply:
I elected to be consistent and just use "python" in the latest version. We can do a project-wide change to "pyhon3" as a follow-up potentially.